### PR TITLE
add docs on running ecs tasks on k3d

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by pinzon — APPROVED** [review]

File: content/en/user-guide/aws/ecs/index.md [hunk 1]
```diff
@@ -338,6 +341,63 @@ services:
...
+After installing `k3d`, you can run the following commands to create a Kubernetes cluster:
+
+{{< command >}}
+$ export NODE_PORT=31566
+$ k3d cluster create ls-cluster -p "4566:$NODE_PORT" --wait --timeout 5m
+{{< / command >}}
+
+### Import LocalStack image
+
+You can import the LocalStack image to the Kubernetes cluster by running the following command:
```
Comment (simonrw) [inline content/en/user-guide/aws/ecs/index.md:362]: This is not needed for most people. We only do it in the pipeline to make sure we don't pull the public image

File: content/en/user-guide/aws/ecs/index.md [hunk 2]
```diff
@@ -338,6 +341,63 @@ services:
...
+    --cluster ls-cluster \
+    --mode direct
+{{< / command >}}
+
+### Install LocalStack in the cluster
+
+You can now install LocalStack in the Kubernetes cluster by using LocalStack's Helm chart. The following command installs LocalStack with the `kubernetes` executor for ECS and sets the `LOCALSTACK_AUTH_TOKEN` environment variable:
+
+{{< command >}}
+$ helm upgrade --install localstack localstack/localstack \
```
Comment (simonrw) [inline content/en/user-guide/aws/ecs/index.md:375]: This can be simplified tbd

**Review by simonrw — CHANGES_REQUESTED** [review]
Thanks for writing this up but there are some simplifications we can make. I will give more info when I am back at my computer

File: content/en/user-guide/aws/ecs/index.md (hunk 2 above)
Comment (simonrw) [reply content/en/user-guide/aws/ecs/index.md:365]: Implemented in bcaa67a9c0700e930bde94a592da32cfd5a10674

**Review by simonrw — APPROVED** [review]
I have implemented my suggestions in bcaa67a9c0700e930bde94a592da32cfd5a10674

File: content/en/user-guide/aws/ecs/index.md [hunk 3]
```diff
@@ -338,6 +341,46 @@ services:
 
 Alternatively, you can download the image from the private registry before using it or employ an [Initialization Hook](https://docs.localstack.cloud/references/init-hooks/) to install the Docker client and use these credentials to download the image.
 
+
+## Running ECS on Kubernetes
+
+LocalStack Enterprise image allows you to run ECS tasks on Kubernetes. The tasks are added to ELB load balancer target groups. You can do so by setting the `ECS_TASK_EXECUTOR` environment variable to `kubernetes` in the LocalStack container.
```
Comment (simonrw) [inline content/en/user-guide/aws/ecs/index.md:347]:
```suggestion
LocalStack Enterprise image allows you to run ECS tasks on Kubernetes. The tasks are added to ELB load balancer target groups.
You can do so by setting the `ECS_TASK_EXECUTOR` environment variable to `kubernetes` in the LocalStack container.
```

File: content/en/user-guide/aws/ecs/index.md [hunk 4]
```diff
@@ -338,6 +341,46 @@ services:
...
+After installing `k3d`, you can run the following commands to create a Kubernetes cluster:
+
+{{< command >}}
+$ export NODE_PORT=31566
+$ k3d cluster create ls-cluster -p "4566:$NODE_PORT" --wait --timeout 5m
+{{< / command >}}
+
+### Install LocalStack in the cluster
+
+You can now install LocalStack in the Kubernetes cluster by using LocalStack's Helm chart. The following command installs LocalStack with the `kubernetes` executor for ECS and sets the `LOCALSTACK_AUTH_TOKEN` environment variable:
```
Comment (simonrw) [inline content/en/user-guide/aws/ecs/index.md:362]:
```suggestion
You can now install LocalStack in the Kubernetes cluster by using LocalStack's Helm chart.
The following command installs LocalStack with the `kubernetes` executor for ECS and sets the `LOCALSTACK_AUTH_TOKEN` environment variable:
```

**Pull request merged**